datumaro/components/dataset.py

```diff
@@ -834,6 +881,16 @@ def flush_changes(self):
     @scoped
     def export(self, save_dir: str, format: Union[str, Type[Extractor]],
             **kwargs) -> None:
+        """
+        Saves the input dataset in some format.
+
+        Args:
+            format (str or Converter) - The desired output format.
```

Comment:
The type here contradicts the function signature:

* the signature has `Extractor` instead of `Converter` (the signature is wrong here).
* the signature asks for a _subtype_ of `Converter` instead of an instance (the docstring is wrong here).

Reply:
> the signature asks for a subtype of Converter instead of an instance (the docstring is wrong here).

It can be both, actually. Fixed